### PR TITLE
Special number in Python

### LINKED ISSUE #6215: Special Number in Python
## 🚀 Feature

A python program to check for special number.

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### PR DESCRIPTION
**Pull request by Khushi-agarwal**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

It is a python program to check whether the given number is special or not.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #6215

### CODE REVIEW

**Comment by Khushi-agarwal** [comment]
@bhav09 @HarshCasper Kindly review it.Thanks.

**Review by ankitaggarwal23 — COMMENTED** [review]
Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

**Comment by Khushi-agarwal** [comment]
@ankitaggarwal23 kindly have a look.Thanks.

**Comment by Khushi-agarwal** [comment]
@ankitaggarwal23 Now the file is updated as per the instructions.

File: Python/math/special.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+if __name__=="__main__":
+    main()
+
+'''
+Time Complexity:O(n)
+Space Complexity:O(1)
+Input/Output:
+Enter a number 111
+The given number is special
+'''
```
Comment (ankitaggarwal23) [inline Python/math/special.py]: Maintain some newline space wherever needed

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Python/math/special.py (hunk 1 above)
Comment (Khushi-agarwal) [reply Python/math/special.py]: > Maintain some newline space wherever needed

Added the space.

**Comment by Khushi-agarwal** [comment]
@ankitaggarwal23 done the changes.

**Review by HarshCasper — APPROVED** [review]

**Comment by siddharth25pandey** [comment]
@Khushi-agarwal Your information is incorrect.
The definition for **Special number** is incorrect & this Program doesn't depicts any Algorithm or logic for the Approach.
Hence **Invalid PR**

**Pull request merged**